node.sh

```diff
@@ -0,0 +1,42 @@
+package: node
+version: "v9.8.0"
+build_requires:
+  - curl
+prefer_system: "(?!slc5)"
+prefer_system_check: |
+  node -h &> /dev/null || { echo "node is missing"; exit 1; }
+  if [[ printf "%d%04d%04d" $(node --version | sed -e 's/v//g; s/\./ /g') -le 900090000 ]]; then echo "higher node version required"; exit 1; fi
+---
+#!/bin/bash
+
+NODEOS=linux
+case $ARCHITECTURE in
+  osx*) NODEOS=darwin ;;
+esac
+
+FILE_NAME="$PKGNAME-$PKGVERSION-$NODEOS-x64"
+TAR_NAME="$FILE_NAME.tar.gz"
+URL="https://nodejs.org/dist/$PKGVERSION/$TAR_NAME"
+curl -O -L $URL
```

Comment:
So, indeed, the `curl` + `tar` + `rsync` can become:

```bash
cd "$INSTALLROOT"
curl -L $URL | tar xzf -
```

In practice you unpack directly into the installation dir, and without even needing the intermediate tarball.

Reply:
(So, no cleanup needed.)

Reply:
Ok done, but in such case I need to move files after untaring.